web/ai_service.py

```
@@ -0,0 +1,141 @@
+import markdown
+from pygments import highlight
+from pygments.lexers import get_lexer_by_name
+from pygments.formatters import HtmlFormatter
+
+class AIService:
+    def __init__(self):
+        self.openai_client = None
+        self.gemini_api_key = None
+        self._initialize_services()
+
+    def _initialize_services(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_initialize_services`

Add return type annotation: `None`

(ANN202)